### DISCUSSION TITLE
v2.10.2

### DISCUSSION THREAD

**Discussion by mienaiyami** [mienaiyami/yomikiru#102] [General]
## Downloads

[![Download setup](https://img.shields.io/badge/Windows%20Setup%20(exe)-Yomikiru--2.10.2--Setup.exe-brightgreen?logo=windows&logoColor=blue)](https://github.com/mienaiyami/yomikiru/releases/download/v2.10.2/Yomikiru-2.10.2-Setup.exe)

[![Download Portable](https://img.shields.io/badge/Windows%20Portable%20(zip)-Yomikiru--win32--2.10.2--Portable.zip-brightgreen?logo=windows&logoColor=blue)](https://github.com/mienaiyami/yomikiru/releases/download/v2.10.2/Yomikiru-win32-2.10.2-Portable.zip)

---
## Note

**Epub reader available**

Linux distribution currently not available as direct download but can be build easily by following this [Build for linux(debian) guide](https://github.com/mienaiyami/yomikiru/blob/master/docs/build_for_linux.md).

---

## Added

- EPUB:
  - Sidelist+TOC.
  - Find in page.
  - Option to load only one chapter at a time to prevent high ram usage(enable from settings).

## Changed

- EPUB: Optimized rendering.

## Fixed

- #101, #99.

---

#### Todo

- EPUB: History+Bookmark.
- Write tests.
- Add ability to change image quality in canvas mode for less RAM usage and performance.

---

**Full Changelog**: https://github.com/mienaiyami/yomikiru/compare/v2.10.1...v2.10.2

<hr /><em>This discussion was created from the release <a href='https://github.com/mienaiyami/yomikiru/releases/tag/v2.10.2'>v2.10.2</a>.</em>

**Comment by zeoint** [comment]
1. Thank you very much for adding epub support and implementing the options that I have asked for.
2. It is now very fluid and uses less ram too. I like the whole Table of contents being in the side list, the same as manga style.
3. Because of it the transition between chapters is seamless.
4. I still have a couple more small requests / issues, I will open them.
5. My heart felt thanks to you.
